### ISSUE TITLE
Request to add break command

### ISSUE THREAD

**Issue by chrismolloy** [bblanchon/ArduinoTrace#8]
It would be useful to have a BREAK() command to "stop" execution and wait for a CR input to continue.

It would stop your trace output to allow you to view it, then continue with a CR  instead of trying to catch it by stopping scrolling.

I prototyped it and it works great!
void BREAK(){
    Serial.print("BREAK  hit enter");
while (!(Serial.available() > 0)){} //wait for input

  int incomingByte = Serial.read();
}

**Comment by ThorsenRune** [comment]
Great idea, how can it be merged into the macros?

**Comment by bblanchon** [comment]
Hi guys,

I'm sorry I made you wait for so long.
A new `BREAK()` macro is available in version 1.2.0.

The implementation is slightly different from what proposed @chrismolloy.
Instead of waiting for any character and letting it in the serial buffer, I decided to wait for `\n` and consume the input.
I made this change because I wanted to remove the extra line-break sent by the Serial Monitor.
I hope it will still fit your needs.

Best regards,
Benoit
